_includes/story.md

```diff
@@ -423,20 +439,20 @@ Concurrency alone is good enough for IO-blocking jobs, but as you saw in a previ
 A common CPU-blocking job is image processing.
 
 - In the context of our scenario let's say an image processing job takes 4 seconds on average.
-- Recall from earlier, we said that we have a large Rails application with 1000 QPS and 1% of that is image processing, which means we have 864,000 image processing jobs per day
-- Multiply 864,000 by 4s and you have 960 hrs worth of processing lined up in a period of 24 hrs, giving us a 1:40 enqueue:processing ratio.
+- Recall from earlier, we said that we have a large Rails application with 1000 QPS and 1% of these queries involve some image processing, which means we have 864,000 image processing jobs per day
```

Comment:
`As we mentioned earlier, we have a large Rails application with 1000 QPS and 1% of these queries involve image processing, which means we have 864,000 image processing jobs per day`